### ISSUE TITLE
4D Render

### ISSUE THREAD

**Issue by thangnd2000** [oppo-us-research/SpacetimeGaussians#78]
Can I create a full range of rotations so I can have a quality output at all angles? for example 360 ​​degrees.

**Comment by lizhan17** [comment]
yes, you can. but i didn't try 360 degrees.  initialation of 360 degree scene may be important. 
and i guess we should increase densify times instead of pruning too many times. 
the original 3dgs works well on 360 degree with many times of densifying.

**Comment by thangnd2000** [comment]
Hi @lizhan17 Is there any way to improve the output quality when the number of rotation angles is very small?

**Comment by lizhan17** [comment]
I think when views are sparse, improving gaussian points or having more views should benefit.
